src/platform/common/utils.ts

```diff
@@ -425,6 +425,10 @@ export class Disposables implements IDisposable {
     constructor(...disposables: IDisposable[]) {
         this.disposables.push(...disposables);
     }
+    protected registerDisposable<T extends IDisposable>(disposable: T): T {
```

Comment:
Making this `register` (like VS Code's `Disposable._register`) or `add` (like VS Code's `DisposableStore.add`) will make things a lot more concise

Reply:
Thats why i couldn't find it, i looked for `register` instead of `_register`